Source/Core/Core/HW/WiimoteEmu/DesiredWiimoteState.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <optional>
+
+#include "Core/HW/WiimoteCommon/WiimoteReport.h"
+#include "Core/HW/WiimoteEmu/Camera.h"
+#include "Core/HW/WiimoteEmu/Extension/DesiredExtensionState.h"
+#include "Core/HW/WiimoteEmu/MotionPlus.h"
+
+namespace WiimoteEmu
+{
+struct DesiredWiimoteState
+{
+  // 1g in Z direction, which is the default returned by an unmoving emulated Wiimote.
+  static constexpr WiimoteCommon::AccelData DEFAULT_ACCELERATION =
+      WiimoteCommon::AccelData({512, 512, 616});
```

Comment:
This is nitpicky, but `ACCEL_ZERO_G` and `ACCEL_ONE_G` (bit shifted left by 2 because the calibration values have 8 bits of precision) could be used to avoid repeat magic numbers.

Reply:
Ah, yeah probably.